tests/plugins.test.ts

```diff
@@ -38,7 +39,7 @@ afterEach(async () => {
 })
 
 test('setupPlugins and runProcessEvent', async () => {
-    getPluginRows.mockReturnValueOnce([plugin60])
+    getPluginRows.mockReturnValueOnce([{ ...plugin60 }])
```

Comment:
it took me more than an hour on Friday night to figure out *this* was the issue with tests! :/ (since it affected plugin60 directly, and wouldn't show up if I simply ran the tests in isolation) - only when things run in sequence did this bork things up. 🤦 .

Reply:
Ooh... that's a nice rabbit hole :D